e2e/plugin-e2e/prometheus/prometheus.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { expect, test } from '@grafana/plugin-e2e';
+
+import { enterCodeEditorQueryExpr, setEditorMode } from './utils';
+
+test('Smoke test: decoupled plugin loads', async ({ createDataSourceConfigPage, page }) => {
```

Comment:
```suggestion
test('Smoke test: config editor loads', async ({ createDataSourceConfigPage, page }) => {
```
Nit, but this test only verifies that the config editor loads. Other views such as the query editor and the variable editor may still not function properly.

Reply:
Great catch! I'll add some more tests for other views.